src/wp-includes/l10n/class-wp-translation-controller.php

```diff
@@ -0,0 +1,410 @@
+<?php
+/**
+ * I18N: WP_Translation_Controller class.
+ *
+ * @package WordPress
+ * @subpackage I18N
+ * @since 6.5.0
+ */
+
+/**
+ * Class WP_Translation_Controller.
+ *
+ * @since 6.5.0
+ */
+class WP_Translation_Controller {
+	/**
+	 * Current locale.
+	 *
+	 * @since 6.5.0
+	 * @var string
+	 */
+	protected $current_locale = 'en_US';
+
+	/**
+	 * Map of loaded translations per locale and text domain.
+	 *
+	 * [ Locale => [ Textdomain => [ .., .. ] ] ]
```

Comment:
what does `..` signify? Is it `...`?

```suggestion
	 * [ Locale => [ Textdomain => [ ..., ... ] ] ]
```

Reply:
It signifies a placeholder, so yes, I guess three dots works as well